.github/workflows/cifuzz.yml

```diff
@@ -1,5 +1,20 @@
 name: CIFuzz
-on: [pull_request]
+on: 
+  push:
+    branches:
+      - master
+    tags:
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
+      - 'OWNERS'
+      - 'MAINTAINERS'
+  pull_request:
+    paths-ignore:
+      - '**.md'
+      - 'docs/**'
```

Comment:
Small question, `**` is the grammer in github action? :) @gy95

Reply:
can match more string than only one `*`
ref: https://docs.github.com/cn/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

Reply:
/hold need some fix

Reply:
Is `**/OWNERS` needed?

Reply:
/hold cancel
fixed